fdbcli/fdbcli.actor.cpp

```diff
@@ -3457,7 +3457,7 @@ ACTOR Future<int> cli(CLIOptions opt, LineNoise* plinenoise) {
 									if (db->apiVersionAtLeast(700))
 										BinaryReader::fromStringRef<ClientWorkerInterface>(
 										    address_interface[tokens[i]].first, IncludeVersion())
-										    .reboot.send(RebootRequest(false, false, timeout_ms));
+										    .reboot.send(RebootRequest(false, false, seconds));
```

Comment:
nit: Can you add a comment to `RebootRequests` to avoid confusion in the future?

Reply:
Done